test/Actions.test.ts

```diff
@@ -0,0 +1,133 @@
+import {ActionTesterInstance} from '../build/types/truffle-types'
+
+const {BN, expectRevert} = require('@openzeppelin/test-helpers')
+
+const ActionTester = artifacts.require('ActionTester.sol')
+
+contract('Actions', ([owner, random]) => {
+  // actionTester mock instance
+  let actionTester: ActionTesterInstance
+  enum ActionType {
+    OpenVault,
+    MintShortOption,
+    BurnShortOption,
+    DepositLongOption,
+    WithdrawLongOption,
+    DepositCollateral,
+    WithdrawCollateral,
+    SettleVault,
+    Exercise,
+    Call,
+  }
+  const ZERO_ADDR = '0x0000000000000000000000000000000000000000'
+
+  before('Deployment', async () => {
+    actionTester = await ActionTester.new({from: owner})
+  })
+
+  describe('Parse Deposit Arguments', () => {
+    it('should not be able to parse a non Deposit action', async () => {
+      const actionType = ActionType.OpenVault
```

Comment:
Use `let` instead of `const` inside `it` blocks.

Reply:
I think it is better practice to use const if we are not changing the value. The linter keeps changing it back when I change it to const